docs/gitrepo-content.md

```diff
@@ -28,6 +28,82 @@ The following files are looked for to determine the how the resources will be de
+
+#### Example File Structure
+```
+driven
+     |___helm
+     |      |__ fleet.yaml
+     |
+     |___simple
+     |      |__ configmap.yaml
+     |      |__ service.yaml
+     |
+     |___kustomize
+            |__ base
+            |      |__ kustomization.yaml
+            |      |__ secret.yaml
+            |
+            |__ overlays
+            |         |__ dev
+            |         |     |__ kustomization.yaml
+            |         |     |__ secret.yaml
+            |         |__ prod
+            |         |     |__ kustomization.yaml
+            |         |     |__ secret.yaml
+            |         |__ test
+            |               |__ kustomization.yaml
+            |               |__ secret.yaml
+            |__ dev.yaml
+            |__ prod.yaml
+            |__ test.yaml
+```
+#### Corresponding GitRepo Definition
+```
+kind: GitRepo
+apiVersion: fleet.cattle.io/v1alpha1
+metadata:
+  name: driven
+  namespace: fleet-local
+spec:
+  repo: https://github.com/0xavi0/fleet-test-data
+  branch: driven-scan-example
+  bundles:
+  - path: driven/helm
+  - path: driven/simple
+  - path: driven/kustomize
+    options: dev.yaml
+  - path: driven/kustomize
+    options: test.yaml
+```
+
+In the example above, the user explicitly defines four Bundles to be generated.
+
+* In the first case, the base directory is specified as `driven/helm`. As shown in the directory structure, this path contains a `fleet.yaml` file, which will be used to configure the Bundle.
+
+* In the second case, the base directory is `driven/simple`, which contains only Kubernetes resource manifests (`configmap.yaml` and `service.yaml`). Since no `fleet.yaml` or options file is specified, Fleet will generate a Bundle using the default behavior—simply packaging all resources found within the directory.
+
+* The third and fourth cases both reference the same base directory: `driven/kustomize`. However, each specifies a different options file (`dev.yaml` and `test.yaml`, respectively). These options files define overlay-specific configuration for each environment (e.g., dev, test) by selecting the appropriate kustomize overlay subdirectories and applying them on top of the shared base.
+Fleet will process these as distinct Bundles, even though they originate from the same base path, because the provided options files point to different configurations.
+
+
+An example of the files used in the third and fourth Bundles would be the following: (These files follow the exact same format as `fleet.yaml`, but since we can now reference them by name, we can use one that best suits our needs)
+```yaml
+namespace: kustomize-dev
+kustomize:
+  dir: "overlays/dev"
+```
+It is important to note that any path defined in these files must be relative to the base directory used when the Bundle was described.
```

Comment:
I think this would deserve a couple of examples, e.g. one that would work and one that would not.